src/metabase/server/middleware/security.clj

```diff
@@ -118,6 +118,11 @@
   (when (and (embed.settings/enable-embedding) (embed.settings/embedding-app-origin))
     (embed.settings/embedding-app-origin)))
 
+(defn- embedding-app-origin-sdk
+  []
+  (when (embed.settings/enable-embedding)
```

Comment:
use a similar logic to `embedding-app-origin`, but we don't need `and` because the result will always have at least a domain `localhost:*` anyway.